src/components/ProjectSearch.js

```diff
@@ -261,10 +261,14 @@ export class ProjectSearch extends Component<Props, State> {
         />
       );
     }
-    if (
-      status === statusType.fetching ||
-      (this.props.query && !results.length)
-    ) {
+    if (status === statusType.fetching) {
+      return (
+        <div className="no-result-msg absolute-center">
```

Comment:
It seems we have a bit of duplicated html here and might want to refactor the code to shave off some lines.

Reply:
Aaha let me see what I can shave off 😉

Reply:
has this already been fixed? *looks for duplicated html

Reply:
@codehag @lukaszsobek 
sorry, no I was super busy last few days. I could take a look only today... :( 
Do you have any specific suggestions?

Reply:
we can probably simplify the code, as it looks like we are accounting for every case. we can add a 
```javascript
if (!this.props.query) return
```
but its probably not necessary.

for the rest we can do:

```javascript
const msg = status === statusType.fetching 
  ? L10N.getStr("loadingText") 
  : L10N.getStr("projectTextSearch.noResults");

return (
  <div className="no-result-msg absolute-center">
    {msg}
  </div>
```